src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,60 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private String manufacturer;
+    private String model;
+    private String type;
+    private int passengersAmount;
 
+    public Car(Engine engine, String manufacturer, String model, String type,
+               int passengersAmount) {
```

Comment:
you can include `Engine` in constructor

Reply:
But I have Engine in the constructor already.

Reply:
Or you mean in main method?